language/move-stdlib/sources/Option.move

```diff
@@ -178,6 +178,21 @@ module Std::Option {
         ensures borrow(t) == e;
     }
 
+    /// Swap the old value inside `t` with `e` and return the old value if any.
+    /// Different from swap(), swap_opt() allows for t not holding a value.
+    public fun swap_opt<Element>(t: &mut Option<Element>, e: Element): Option<Element> {
```

Comment:
@tzakian do we have a naming convention for these sorts of things? I know we ran into something similar with Strings

Reply:
I don't believe so. This one is doing a kind of weird operation. Another option for a name could be `swap_or_fill`?

Reply:
@tzakian Messed up my github branch. Had to create another PR with swap_or_fill as the name: https://github.com/diem/diem/pull/10018